### DISCUSSION TITLE
Nordigen iban field invalid from my account - possible to insert a custom processor?

### DISCUSSION THREAD

**Discussion by sknudsen** [firefly-iii/firefly-iii#7755] [Firefly III]
Hi,

The json transaction data from Nordigen for my bank account include this field (d indicating a single digit): 
```
"creditorAccount": {
                    "iban": "dddddddddd"
                },
```
Obviously, this is an invalid iban specification. 

However, I would like to be able to insert a custom processor to make the Nordigen output match my asset accounts. Alternatively, I would need to insert a few lines of code in data-importer/Services/Nordigen/Model/Transaction.php, but that seems to be the wrong way to solve this given that the code behaves as it should. Also, it would be a pain to manage.

Is there another way I could do this?

Thanks

**Comment by JC5** [comment]
Mmm, invalid IBANs, that's a new one :o.

I'm not really sure. There is no room for custom processors in the data importer, that's a huge pita. You could always fork the importer and edit the necessary lines, but that may be a bit much as well.

You could also contact Nordigen and see if they can filter out these IBANs. But as far as I know, they usually passthrough whatever they get.
